lib/modules/datasource/docker/common.ts

```diff
@@ -144,23 +146,20 @@ export async function getAuthHeaders(
     ) {
       scope = authenticateHeader.params.scope;
     }
+    authUrl.searchParams.append('scope', scope);
```

Comment:
better use an if else here? than we can remove the `scope` variable and i think it's more readable.